cli/tests/functional/src/test/resources/archetypev2/common/files/README.md

```diff
@@ -0,0 +1,3 @@
+# My Readme file
```

Comment:
Remember that you are not testing the actual Helidon archetypes, so this archetype's purpose is to run the cli tests. Please trim it down so that it only contains things needed by the tests. 

Do you need README.md and app.yaml ; do you need any actual Helidon based code at all ? 
If you need to simulate a server, can't you do it with just the JDK ?

Reply:
I need one dependency to reproduce the devloop issue, and I removed everything else.